repoze/sphinx/autointerface.py

```diff
@@ -98,16 +107,25 @@ def setup(app):
 
     from sphinx.domains import ObjType
 
+    try:
+        domains = app.domains
+    except AttributeError:
+        domains = app.registry.domains
```

Comment:
Would prefer to have the "current" usage first, falling back to the older one. E.g.:

```python

    try:
        domains = app.registry.domains
    except AttributeError:
        domains = app.domains
```